public/js/header-9.js

```diff
@@ -0,0 +1,14 @@
+let navToggle = document.querySelector(".nav__toggle");
```

Comment:
Similar with `demo.js`, let's see if there is a better name than `header-9.js`. If not, still `header.js` seems more appropriate :-)